Podcast/ReleaseCard.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ReleaseCard.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 2/22/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+//when a series releases an episode 
+class ReleaseCard: EpisodeCard {
+    
+    var seriesID: Int
+    var seriesImageURL: URL?
+    var seriesImage: UIImage?
+    
+    init(episodeID: Int, episodeTitle: String = "", dateCreated: Date = Date(), descriptionText: String = "Not avaliable", smallArtworkImage: UIImage = #imageLiteral(resourceName: "filler_image"), time: Double = 0.0, nRecommended: Int = 0, tags: [String] = [], seriesTitle: String = "", seriesID: Int = 0, isBookmarked: Bool = false, isRecommended: Bool = false, seriesImageURL: URL, seriesImage: UIImage = #imageLiteral(resourceName: "filler_image")) {
+        
+        self.seriesID = seriesID
+        self.seriesImageURL = seriesImageURL
+        self.seriesImage = seriesImage
```

Comment:
you wouldn't pass in both the image url and the image, you probably just need the imageURL as an argument